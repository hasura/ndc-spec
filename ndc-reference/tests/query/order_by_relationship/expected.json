[
  {
    "rows": [
      {
        "id": 2,
        "title": "Why Functional Programming Matters",
        "author": {
          "rows": [
            {
              "first_name": "John",
              "last_name": "Hughes"
            }
          ]
        }
      },
      {
        "id": 3,
        "title": "The Design And Implementation Of Programming Languages",
        "author": {
          "rows": [
            {
              "first_name": "John",
              "last_name": "Hughes"
            }
          ]
        }
      },
      {
        "id": 1,
        "title": "The Next 700 Programming Languages",
        "author": {
          "rows": [
            {
              "first_name": "Peter",
              "last_name": "Landin"
            }
          ]
        }
      }
    ]
  }
]