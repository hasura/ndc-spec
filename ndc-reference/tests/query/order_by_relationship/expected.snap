---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/order_by_relationship/request.json
---
[
  {
    "rows": [
      {
        "id": 2,
        "title": "Why Functional Programming Matters",
        "author": {
          "rows": [
            {
              "first_name": "John",
              "last_name": "Hughes"
            }
          ]
        }
      },
      {
        "id": 3,
        "title": "The Design And Implementation Of Programming Languages",
        "author": {
          "rows": [
            {
              "first_name": "John",
              "last_name": "Hughes"
            }
          ]
        }
      },
      {
        "id": 1,
        "title": "The Next 700 Programming Languages",
        "author": {
          "rows": [
            {
              "first_name": "Peter",
              "last_name": "Landin"
            }
          ]
        }
      }
    ]
  }
]
